File: test/qr.dart
import 'package:qr/qr.dart';

void main() {
  final qr = QrCode.fromData(data: 'unsandbox-qr-ok', errorCorrectLevel: QrErrorCorrectLevel.M);
  print('QR:unsandbox-qr-ok:ROWS:${qr.moduleCount}');
}

File: test/qr.cs
using QRCoder;

class Program {
    static void Main() {
        var gen = new QRCodeGenerator();
        var data = gen.CreateQrCode("unsandbox-qr-ok", QRCodeGenerator.ECCLevel.M);
        var rows = data.ModuleMatrix.Count;
        Console.WriteLine($"QR:unsandbox-qr-ok:ROWS:{rows}");
    }
}
